NEWS.md

```diff
@@ -1,3 +1,8 @@
+# sidora 2.0.4.9000
```

Comment:
I'm not a big fan of Hadley's `.9000` version suffixes. What do you want to express here?

Reply:
Just that it's in `dev` version (not in a releasE). I thought `.9000` was general R/CRAN standard,  can put `dev` or something else as you wish

Reply:
What I'm trying to say is: We don't need any dev versions. I suggest we just do [semantic versioning](https://semver.org/) without any tshatshke.

Reply:
Oh I see. OK, how does this work with actual releases then? Or you think we ignore that entirely?

Reply:
The GitHub releases have no real relevance in this project, I fear. Typically users install the latest version directly from the master branch anyway. I suggest we do the GitHub releases whenever we reached a perceived milestone.

Reply:
kk